src/screens/Topics/index.js

```diff
@@ -187,7 +189,9 @@ const Topics = () => {
   );
 };
 const styles = StyleSheet.create({
-  scrollViewStyle: {marginBottom: 100},
+  scrollViewStyle: (bottom) => ({
+    marginBottom: 100 - bottom
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The calculation for `marginBottom` seems to be incorrect. If `bottom` is the safe area inset at the bottom of the screen, then subtracting it from 100 might not give the desired margin. The `marginBottom` should be equal to `bottom` if you want the ScrollView content to be above the safe area.
```diff
- marginBottom: 100 - bottom
+ marginBottom: bottom
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
need to add 100 because of button bottom height is 100